bedrock/mozorg/templates/mozorg/contribute/signup-alternate.html

```diff
@@ -0,0 +1,123 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/contribute-base.html" %}
+
+{% add_lang_files "mozorg/contribute/index" %}
+
+{% block body_id %}inquiry{% endblock %}
+
+{% block page_title %}{{_('Get Involved: Volunteer Opportunities at Mozilla')}}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'contribute-signup' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if LANG == 'en-US' and waffle.switch(‘contribute-signup-optimizely') %}
```

Comment:
There's a fancy single quote at the start of the waffle switch name that's throwing errors.